examples/verif-manual/vm-014.py

```diff
@@ -0,0 +1,162 @@
+mapdl.egen(4, 1, 1)
+
+# Apply load and boundary conditions
+mapdl.d(1, "ALL")  # Fix all degrees of freedom for node 1
+mapdl.f(5, "FY", -4000)  # Apply a negative force FY to node 5
+mapdl.dsym("SYMM", "Z")  # Apply symmetry boundary condition in Z-direction
+
+# select all entities
+mapdl.allsel()
+# element plot
+mapdl.eplot()
+
+# Finish the pre-processing processor
+mapdl.finish()
+
+# Enter the solution processor to define solution controls
+mapdl.slashsolu()
+
+# Activate large deflections
+mapdl.nlgeom("ON")
+
+# Set convergence tolerances
+mapdl.cnvtol("F", "", 1e-4)
+mapdl.cnvtol("M", "", 1e-4)
+
+# Controls the solution printout
+mapdl.outpr("", "LAST")
+
+# Run non-interactive commands
+with mapdl.non_interactive:
+    mapdl.run("/OUT,SCRATCH")  # redirects solver output to a file named "SCRATCH"
+    mapdl.solve()  # starts a solution
+    mapdl.finish()  # exists solution processor
+
+    mapdl.post1()  # enter post-processing processor
+    mapdl.run("/OUT")  # redirects output to the default system output file
+    mapdl.gopr()  # reactivates suppressed printout
+
+# Retrieve nodal deflection and section stresses
+mapdl.run("END_NODE = NODE (0,120/2,0)")
+deform = mapdl.get("DEF", "NODE", "END_NODE", "U", "X")  # Nodal deflection
+Strss_Tens = mapdl.get(
+    "STS_TENS", "SECR", 1, "S", "X", "MAX"
+)  # Maximum section tensile stress
+Strss_Comp = mapdl.get(
+    "STS_COMP", "SECR", 1, "S", "X", "MIN"
+)  # Minimum section compressive stress
+
+# Defines an array parameter and its dimensions of output quantities.
+mapdl.dim("LABEL", "CHAR", 3, 2)
+mapdl.dim("VALUE", "", 3, 3)
+
+# Set labels for output variables
+mapdl.run("LABEL(1,1) = 'DEFLECTI','STRSS_TE','STRSS_CO'")
+mapdl.run("LABEL(1,2) = 'ON (in) ','NS (psi)','MP (psi)'")
+
+# Fill the VALUE table with data
+mapdl.vfill("VALUE(1", "1)", "DATA", 0.1086, 1803.63, -2394.53)
+mapdl.vfill("VALUE(1", "2)", "DATA", "ABS(DEF)", "STS_TENS", "STS_COMP")
+mapdl.vfill(
+    "VALUE(1",
+    "3)",
+    "DATA",
+    "ABS(DEF/.1086 )",
+    "ABS(STS_TENS/1803.63)",
+    "ABS(STS_COMP/2394.53)",
+)
+
+# Run non-interactive commands
+with mapdl.non_interactive:
+    mapdl.run("/OUT,vm14,vrt")  # Output to a file named vm14.vrt
+    mapdl.com("------------------- VM14 RESULTS COMPARISON ---------------------")
+    mapdl.com("")
+    mapdl.com("|   TARGET   |   Mechanical APDL   |   RATIO")
+    mapdl.com("")
+    mapdl.run("*VWRITE,LABEL(1,1),LABEL(1,2),VALUE(1,1),VALUE(1,2),VALUE(1,3)")
+    mapdl.run("(1X,A8,A8,'   ',F10.4,'  ',F12.4,'   ',1F15.3)")
+    mapdl.com("-----------------------------------------------------------------")
+    # redirects output to the default system output file
+    mapdl.run("/OUT")
+    # reactivates suppressed printout
+    mapdl.gopr()
+
+# Get the mapdl temporary working directory
+vrt_file_path = os.path.join(mapdl.directory, "vm14.vrt")
+
+# read the vm14.vrt file to print the results
+f = open(vrt_file_path, "r")
+for x in f:
+    print(x)
```

Comment:
Using the same syntax as for `vm13.py` would be great. After removing this section, it would look like this:
```
message = f"""
------------------- VM14 RESULTS COMPARISON ---------------------
   RESULT      |  TARGET     |   ANSYS          |   RATIO
   name             target        {ans_result:.5f}  {abs(ans_result/target):.5f}
   name             target        {ans_result:.5f}  {abs(ans_result/target):.5f}
-----------------------------------------------------------------
print(f)
```

Reply:
@clatapie : Initially, I tried the same approach as in vm13.py but below variables has string attached with the numeric values. Hence I keep the codes as we have.

Strss_Tens: 1807.34485    
   AT ELEMENT NODE I

Strss_Comp: -2396.00049
   AT ELEMENT NODE I